pkg/machine/wsl/util_windows.go

```diff
@@ -0,0 +1,338 @@
+package wsl
+
+import (
+	"encoding/base64"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+	"syscall"
+	"unicode/utf16"
+	"unsafe"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"golang.org/x/sys/windows"
+	"golang.org/x/sys/windows/registry"
+
+	"github.com/containers/storage/pkg/homedir"
+)
+
+//nolint
+type SHELLEXECUTEINFO struct {
```

Comment:
nit: I am not sure about type naming scheme but should this be `ShellExecuteInfo`

Reply:
There's a comment on line 57 that talks about this, but it would have been better higher in the source or repeated. The rationale is to follow the pattern in the syscall package, to mirror the win32 api style to make it easier to associate the calls with the exact win32 function. I'm happy to change it if you all prefer though